src/js/index.js

```diff
@@ -13,6 +13,74 @@ const navBookmark = document.querySelector('[data-js=navBookmark]')
 const navCreate = document.querySelector('[data-js=navCreate]')
 const navSettings = document.querySelector('[data-js=navSettings]')
 
+// Get main sections (^= pages)
+const pages = document.querySelectorAll('.page')
+
+// Get each main section (^= page)
+const createPage = document.querySelector('.create-page')
+const homePage = document.querySelector('.app-grid')
+
+// Get Header Heading
+const headerHeading = document.querySelector('h1')
+
+// Get form
+const form = document.querySelector('form')
+
+// Get textareas
+const questionArea = document.querySelector('.create-page__form__add-question')
+const answerArea = document.querySelector('.create-page__form__add-answer')
+const tagsArea = document.querySelector('.create-page__form__add-tags')
+
+// Get character counters
+const questionCounter = document.querySelector(
+  '.create-page__form__add-question__counter'
+)
+const answerCounter = document.querySelector(
+  '.create-page__form__add-answer__counter'
+)
+const tagCounter = document.querySelector(
+  '.create-page__form__add-tags__counter'
+)
+
+// Get button
+const formButton = document.querySelector('.create-page__form__button')
+
+/* ALPHA TESTING 
```

Comment:
Please move experimental code to separate branches. You can merge or delete them later, but they would not spam your production code.